sdk/java-sdk/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala

```diff
@@ -153,7 +153,7 @@ class DiscoveryImpl(
               name,
               Component.ComponentSettings.Entity(
                 EntitySettings(
-                  service.entityType,
+                  service.workflowName,
```

Comment:
leftover?

Reply:
That was certainly a too enthusiastic refactoring. 😄 

It's also not correct to call this entityType. Actions and Views are also Services, but not Entities. For Views, we are using the `viewId`, for action we are simply using the descriptor name. 

This is internal API. We can refactor it without problem. Maybe `serviceId` or `serviceName`?